ej2-angular/tooltip/api-tooltipModel.md

```diff
@@ -14,29 +14,29 @@ Interface for a class Tooltip
 
 ## Properties
 
-### afterClose [`EmitType<TooltipEventArgs>`](https://ej2.syncfusion.com/angular/documentation/api-tooltipEventArgs.html)
+### afterClose [`EmitType<TooltipEventArgs>`](https://ej2.syncfusion.com/documentation/api/tooltip/tooltipEventArgs/)
```

Comment:
This is angular platform not javascript. Why have you updated JavaScript link? Check in all places